Lombiq.Hosting.Tenants.EmailQuotaManagement/Services/EmailQuotaService.cs

```diff
@@ -19,54 +19,38 @@
 
 namespace Lombiq.Hosting.Tenants.EmailQuotaManagement.Services;
 
-public class EmailQuotaService : IEmailQuotaService
+public class EmailQuotaService(
+    ISession session,
+    IOptions<EmailQuotaOptions> emailQuotaOptions,
+    IShellConfiguration shellConfiguration,
+    IOptions<SmtpSettings> smtpOptions,
+    IClock clock,
+    IRoleService roleService,
+    UserManager<IUser> userManager) : IEmailQuotaService
 {
-    private readonly ISession _session;
-    private readonly EmailQuotaOptions _emailQuotaOptions;
-    private readonly IShellConfiguration _shellConfiguration;
-    private readonly SmtpSettings _smtpOptions;
-    private readonly IClock _clock;
-    private readonly IRoleService _roleService;
-    private readonly UserManager<IUser> _userManager;
-
-    public EmailQuotaService(
-        ISession session,
-        IOptions<EmailQuotaOptions> emailQuotaOptions,
-        IShellConfiguration shellConfiguration,
-        IOptions<SmtpSettings> smtpOptions,
-        IClock clock,
-        IRoleService roleService,
-        UserManager<IUser> userManager)
-    {
-        _session = session;
-        _emailQuotaOptions = emailQuotaOptions.Value;
-        _shellConfiguration = shellConfiguration;
-        _smtpOptions = smtpOptions.Value;
-        _clock = clock;
-        _roleService = roleService;
-        _userManager = userManager;
-    }
+    private readonly EmailQuotaOptions _emailQuotaOptions = emailQuotaOptions.Value;
+    private readonly SmtpSettings _smtpOptions = smtpOptions.Value;
 
     public async Task<IEnumerable<string>> GetUserEmailsForEmailReminderAsync()
     {
         // Get users with site owner permission.
-        var roles = await _roleService.GetRolesAsync();
+        var roles = await roleService.GetRolesAsync();
```

Comment:
Keep using underscored named for private fields everywhere.

Reply:
That would have an affair with https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1313.md.
Also it would not be compatible with VS built in convert to primary constructor feature.
Also this would mean to change like 3000 private field back to _ version because VS removed them.
We only needed _ version for the sheer reason to differentiate between the injection and the private field.
So because of these I would not recommend switching back to that. Also they look wierd in parameters.

Reply:
I resolve this thus this is not a question anymore.